src/index.css

```diff
@@ -84,27 +84,29 @@ code {
   position: absolute;
   z-index: 1;
   bottom: 42px;
-  left: calc(50% - (3 / 2) * (32px + 2 * 15px));
```

Comment:
You should change this to `calc(50% - (3/2) * (16px + 2*8.5px))` to make sure it is centered (otherwise it puts the left edge in the center)